.github/workflows/checks.yml

```diff
@@ -1,27 +0,0 @@
-on: [pull_request]
-name: checks
-env:
-  GO111MODULE: on
-
-jobs:
-  linter-check:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: golangci-lint
-        uses: golangci/golangci-lint-action@v2
-        with:
-          version: 'latest'
-          args: --config ./golangci.yml
-
-  vet-check:
```

Comment:
Why the removal of the vet-check action?

Reply:
I think it is still run as part of `golangci-lint`.